src/store/app.ts

```diff
@@ -37,24 +29,22 @@ export type Metrics = {
   totalStableProvided: Amount;
 };
 
-// TODO: Support multiple anchors.
-export const metricsAtom = atom<Metrics | null>(null);
+// A map of anchor brand petnames to their instances' metrics.
```

Comment:
:+1 commenting

consider jsdoc so that when someone is hovering over the symbol in use they get the comment too.
```suggestion
/** A map of anchor brand petnames to their instances' metrics. */
```